stable/openvpn/templates/config-openvpn.yaml

```diff
@@ -78,6 +114,7 @@ data:
       sed 's|OVPN_K8S_SEARCH|'"${FORMATTED_SEARCH}"'|' -i /etc/openvpn/openvpn.conf
       sed 's|OVPN_K8S_DNS|'"${DNS}"'|' -i /etc/openvpn/openvpn.conf
       sed 's|NETWORK|'"${NETWORK}"'|' -i /etc/openvpn/openvpn.conf
+      sed 's|NETMASK|'"${NETMASK}"'|' -i /etc/openvpn/openvpn.conf
```

Comment:
maybe I'm missing something here, but I'm guessing we want to put the NETMASK in the openvpn.conf file somewhere? I don't see `NETMASK` in the openvpn.conf file as a placeholder at the moment - are you using `.Values.openvpn.OVPN_SUBNET` in its place?

Reply:
db363d6 Well spotted, embarrasingly my testing didn't catch that as the helm upgrade I performed didn't rewrite the openvpn.conf file which I had manually edited to contain the correction in the mentioned commit.